### ISSUE TITLE
Proper Validation display on respective fields (User Registration)

### ISSUE THREAD

**Issue by deleenananaa** [Tenzing07/One-Byte-Food#6]
To enhance user experience and accessibility, validations and error handling for user registration should be displayed in a popup or dialogue box within the system. The bug is shown in the screenshot below:
![Screenshot 2024-04-17 113715](https://github.com/Tenzing07/One-Byte-Food/assets/119421004/c87fa2d3-3473-42c9-af20-67cf8857cc98)

**Comment by AvimanShahi** [comment]
![image](https://github.com/Tenzing07/One-Byte-Food/assets/119503363/b54f69c1-ea07-40d0-a2a9-b13ac6811bcf)
I have completed task you have assigned to me now the pop will come when the field is empty
